fedimint-cli/Cargo.toml

```diff
@@ -1,12 +1,13 @@
 [package]
 name = "fedimint-cli"
+edition = { workspace = true }
```

Comment:
`edition.workspace = true` maybe

Reply:
That's a nice improvement, huh.

I tried it (`sed -i -e 's/ = { workspace = true }/.workspace = true/g' **.toml` in Fish) and unfortunately `cargo-sort` for some reason can't part it :/):

```
❯ j cargo-sort-check
cargo sort -w -g --order package,features,bin,lib,test,bench,dependencies,dev-dependencies,build-dependencies --check
Checking fedimint...
Checking aead...
thread 'main' panicked at src/sort.rs:113:46:
called `Result::unwrap()` on an `Err` value: TomlError { message: "TOML parse error at line 3, column 8\n  |\n3 | version.workspace = true\n  |        ^\nUnexpected `.`\nExpected `=`\n" }
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
error: Recipe `cargo-sort-check` failed on line 266 with exit code 101
```